README.md

```diff
@@ -16,6 +16,8 @@ BitOps is an automated [orchestrator](docs/about.md) for deployment tools using
 
 It leverages a way to describe infrastructure for many environments and IaC tools called an [Operations Repository](docs/operations-repo-structure.md).
 
+[![BitOps Intro Video](docs/assets/images/bitops-intro-video.png)](https://youtu.be/BiytYu3EefY)
```

Comment:
you can do this like: https://github.com/bitovi/github-actions-deploy-docker-to-ec2/blob/main/README.md#getting-started-intro-video

```
[![Getting Started - Youtube](https://img.youtube.com/vi/YOUTUBE_VIDEO_ID/0.jpg)](https://www.youtube.com/watch?v=YOUTUBE_VIDEO_ID)
```

this will use the image that youtube uses and link it to the youtube page.  so we don't have to check in a cover screen.

Reply:
+adding a "play" overlay for the image so it's more intuitive for the users.

<img src="https://user-images.githubusercontent.com/1533818/213275563-7639750c-26b2-47c3-b429-360731a28ae5.png" width="100"/>

Reply:
Updated the image here https://github.com/bitovi/bitops/pull/386/commits/12fda6334c8a69f03988ac37adfd54c25d59fe68